package.json

```diff
@@ -20,6 +20,7 @@
     "@actions/core": "^1.11.1",
     "@actions/github": "^6.0.0",
     "@ai-sdk/anthropic": "^0.0.56",
+    "@ai-sdk/google": "^0.0.55",
```

Comment:
The `@ai-sdk/google` package is using a caret version range (`^0.0.55`). For a new integration, especially with an AI service, it's recommended to pin to an exact version (`0.0.55`) to ensure consistent behavior. This is particularly important since the package is still in early versions (0.x.x) where breaking changes are more likely.